conan/tools/cmake/cmakedeps2/target_configuration.py

```diff
@@ -380,7 +396,20 @@ def _template(self):
         {% if lib_info.get("system_libs") %}
         target_link_libraries({{lib}} INTERFACE {{lib_info["system_libs"]}})
         {% endif %}
-
+        {% if lib_info.get("frameworks") %}
+        set_property(TARGET {{lib}} PROPERTY INTERFACE_LINK_LIBRARIES "{{config_wrapper(config, lib_info["frameworks"])}}")
+        {% endif %}
+        {% if lib_info.get("package_framework") %}
+        set_target_properties({{lib}} PROPERTIES
```

Comment:
Isn't this the ``else`` of the ``if(CMAKE_VERSION VERSION_LESS "3.24")``?
Maybe we want to force CMake 3.24 at least? We can do that.